locales/en.json

```diff
@@ -2,12 +2,12 @@
     "about": {
         "creatorA": "This site was created and maintained by me, LaShawn Toyoda! If you have any trouble using the site or come across incorrect or outdated information, please contact me through Twitter {0}",
         "creatorQ": "Who created this website?",
-        "supportA": "The best way to help is by submitting any cancellation waiting lists you come across to the database. If you're feeling extra generous, please buy me a kofi!",
-        "supportQ": "How can I help support the database?",
-        "trustA": "All of the links lead to official resources such as clinic and government websites. Please read them thoroughly before registering your personal information with them.",
+        "supportA": "If you'd like to be part of the Find a Doc community, <a href='https://github.com/ourjapanlife'>join us over on GitHub</a> to see what we could use your help with!",
```

Comment:
Does this work? I think we need to use interpolated components:

https://kazupon.github.io/vue-i18n/guide/interpolation.html#basic-usage

Reply:
You're 100% right Ann. I used this previously for our disclaimer here:
https://github.com/ourjapanlife/findadoc-frontend/blob/main/components/Disclaimer.vue#L3

It works, but we do need to be vigilant about making 100% sure when we're injecting HTML.

Reply:
I checked that it does work. I got the idea from the Disclaimer code @renderf0x wrote.

Reply:
Ahhh cool. Let's merge, and I may submit a small PR later to separate the code from the text